README.md

```diff
@@ -48,29 +76,32 @@ The terms used to reference different varieties or seasons of maize/wheat has be
 | [Mexico](data_preparation/crop_statistics_MX/README.md)| white/yellow corn| -|
 | [USA](data_preparation/crop_statistics_US/README.md)| grain corn | winter wheat|
 
-
 ### Getting started
+
 `cybench` is an open source python library to load CY-Bench dataset and run the CY-Bench tasks.
 
 #### Installation
+
 ```
 git clone https://github.com/BigDataWUR/AgML-crop-yield-forecasting
 ```
 
 #### Requirements
+
 The benchmark results were produced in the following test environment:
 
 ```
-Operating system:
-CPU:
-memory (RAM):
-disk storage:
-GPU:
+Operating system: Ubuntu 18.04
+CPU: Intel Xeon Gold 6448Y (32 Cores)
+memory (RAM): 256GB
+disk storage: 2TB
+GPU: NVIDIA RTX A6000
 ```
 
 **Software requirements**: Python 3.10, scikit-learn 1.3.2, PyTorch 2.1.1+cpu.
```

Comment:
@hbja : Could you also specify these? Thanks.

Reply:
Updated!